app/views/menu/_menu_fandoms.html.erb

```diff
@@ -1,10 +1,12 @@
-<ul class="menu" role="menu">
-  <li><%= link_to ts('All Fandoms', key: 'header.fandom'), media_path %></li>
-  <% cache "menu-fandoms-version4", skip_digest: true do %>
-    <% Media.for_menu.each do |medium| %>
-      <% unless medium.id.nil? %>
-        <li id="medium_<%= medium.id %>"><%= link_to ts("#{medium.name}", key: 'header.fandom'), medium_fandoms_path(medium) %></li>
+<div role="navigation" aria-label="<%= t(".media") %>">
+  <ul class="menu">
+    <li><%= link_to t(".all_fandoms"), media_path %></li>
+    <% cache "menu-fandoms-version4", skip_digest: true do %>
+      <% Media.for_menu.each do |medium| %>
+        <% unless medium.id.nil? %>
+          <li id="medium_<%= medium.id %>"><%= link_to t(".#{medium.translation_key}"), medium_fandoms_path(medium) %></li>
```

Comment:
You can ignore this comment; it's observational rather than something you need to action.

This poses a thrilling question for future!us: media are tags. Should we be translating tag names?

It makes sense here because it's site navigation more than a tag, imo, but will we want to show the localized media name, e.g., in the Parent Tags section on a [fandom's tag landing page](https://archiveofourown.org/tags/Broadchurch)? 

I kind of feel like that page should list the tag as-is (because I doubt we'll translate fandoms in the Parent Tags section, e.g., Good Omens (TV) on [Crowley's landing page](https://archiveofourown.org/tags/Crowley%20(Good%20Omens))), how important is it that the site navigation and the parent tags match? 🤔